<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>439</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TasksQ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #fff;
}




QStatusBar{
	background-color: #f5f5f5;

}


QToolBar{
	border:none;
	background-color: #f5f5f5;

margin: 0;
}

QToolButton{
	width: 100%;

}

QToolButton:hover{
background-color: rgba(0,0,0,.3);

}


QListWidget{
	font-size: 16px;
	line-height: 24px;

}
QTableWidget{
	background-color: #c7c7c7;
}



QMenuBar {
    background-color: #c7c7c7;
	padding: 4px 0;
}

QMenuBar::item {
    spacing: 4px; /* spacing between menu bar items */
    padding: 1px 4px;
    background: transparent;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #a8a8a8;
}

QMenuBar::item:pressed {
    background: #888888;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="listPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="sceneRect">
           <rectf>
            <x>0.000000000000000</x>
            <y>0.000000000000000</y>
            <width>21.000000000000000</width>
            <height>4.000000000000000</height>
           </rectf>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="editPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Start date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>End date</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
          </property>
          <property name="centerButtons">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="editPageEndDateInput"/>
        </item>
        <item row="1" column="4">
         <widget class="QTimeEdit" name="editPageStartTimeInput"/>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Start time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="editPageStartDateInput"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editPageNameInput"/>
        </item>
        <item row="3" column="4">
         <widget class="QTimeEdit" name="editPageEndTimeInput"/>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>End time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QComboBox" name="editPageTypeInput"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuTasks">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionReport_bug"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="actionAbout_TaskMaster"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QMenu" name="menuGroup_By">
     <property name="title">
      <string>Group By</string>
     </property>
     <addaction name="actionName"/>
     <addaction name="actionTime"/>
     <addaction name="actionDay"/>
     <addaction name="actionWeek"/>
     <addaction name="actionMonth"/>
    </widget>
    <addaction name="menuGroup_By"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuTasks"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="icon">
    <iconset theme="1" resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_add_black_24dp_2x.png</normaloff>
     <normalon>icons/ic_add_box_black_24dp_2x.png</normalon>:/icons/icons/ic_add_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_edit_black_24dp_2x.png</normaloff>:/icons/icons/ic_edit_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_remove_circle_outline_black_24dp_2x.png</normaloff>:/icons/icons/ic_remove_circle_outline_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Rem</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_help_outline_black_24dp_1x.png</normaloff>:/icons/icons/ic_help_outline_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_update_black_24dp_1x.png</normaloff>:/icons/icons/ic_update_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionReport_bug">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_bug_report_black_24dp_1x.png</normaloff>:/icons/icons/ic_bug_report_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Report  Bug</string>
   </property>
  </action>
  <action name="actionAbout_TaskMaster">
   <property name="text">
    <string>About TaskMaster</string>
   </property>
  </action>
  <action name="actionDay">
   <property name="text">
    <string>Day</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actionWeek">
   <property name="text">
    <string>Week</string>
   </property>
  </action>
  <action name="actionMonth">
   <property name="text">
    <string>Month</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_undo_black_24dp_1x.png</normaloff>:/icons/icons/ic_undo_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="iconsressources.qrc">
     <normaloff>:/icons/icons/ic_redo_black_24dp_1x.png</normaloff>:/icons/icons/ic_redo_black_24dp_1x.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconsressources.qrc"/>
 </resources>
 <connections/>
</ui>
